Makefile

```diff
@@ -1,34 +1,37 @@
 # Deploy all the lambdas
-deploy: setup
-	sam package \
-        --template-file template.yaml \
-        --output-template-file .stack.yaml \
-        --s3-bucket bref-benchmarks
-	sam deploy \
+deploy: build package
+	cd .aws-sam/build \
```

Comment:
Any reason to change the build? It sounds more complex now with a temp directory + using `sam build`, I'd rather avoid complexity if it's not necessary.

Reply:
I understand. This is mainly because each app is now responsible of its own packaging recipe. This is kind of dependency inversion and fine tuned artifact generation. By the way even in nodejs world for example it's usual to have a dist or build directory in which the artifact to deploy is generated, it's not shocking me.

Finally, this is especially handy for the [Symfony app](https://github.com/brefphp/benchmarks/pull/3/files#diff-84f5846fa3683d8d1c794c9eb49072acR1-R9) which has some specific optimizations that help making the benchmark results realistic. This is also a good demonstration of packaging steps to respect in order to deploy such kind of application.